### ISSUE TITLE
Album has no cover, but song shows a cover when playing.

### ISSUE THREAD

**Issue by digimezzo** [digimezzo/dopamine-windows#384]
Why is [this](https://i.gyazo.com/753c9dcd8378622fbaad634620d18955.png) okay ?
Why not change the code to use the song cover if it has no album cover ? only 10% of my music has album covers and no one wants to edit all of his audio files, it looks like a simple if else problem to me.

**Comment by Magentaize** [comment]
The most clearly reason is that he doesn't set the correct album name.

**Comment by digimezzo** [comment]
It's indeed a simple if else problem which is performing as intended. Your files have covers, which are displayed on playback. However, your file for artist AAA doesn't have a album title. So Dopamine has no idea in which album it should classify it. So Dopamine chooses to put the song in an album which is caused "Unknown Album". this  "Unknown Album" will contain all files which don't have an album title set. So it is not allowed by Dopamine to set an album cover for that album. Because it can contain files which have different album covers. Which one to choose to use as an album cover then?

Long story short: fill in the album title tag. Dopamine will then create a separate album for that song, and provide the correct album art.
